.github/workflows/testing.yml

```diff
@@ -198,7 +198,14 @@ jobs:
         if: matrix.tarantool != 'brew' && matrix.tarantool != 'master'
 
       - name: Install tarantool build dependencies
-        run: brew install autoconf automake libtool
+        run: |
+          brew install autoconf automake libtool openssl@1.1
+          brew link --force openssl@1.1
+          export PATH="/usr/local/opt/openssl@1.1/bin:$PATH"
+          export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
+          export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"
+          export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig"
+          export OPENSSL_ROOT_DIR="/usr/local/opt/openssl@1.1"
```

Comment:
AFAIU, those exports are not seen on a next step. If they're actually necessary, use [GITHUB_ENV](https://docs.github.com/en/actions/learn-github-actions/workflow-commands-for-github-actions#setting-an-environment-variable).

Reply:
Sure, these exports are not necessary. I export all needed flags with calling CMake below. Fixed.